docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Spryker now requires PHP 8.0 as the minimum version for all projects
+description: In the coming weeks, Spryker will release a new version of its demo shops requiring PHP 8.0 as the minimum version - support for the PHP 7.4 version will be stopped. Spryker's new module releases will only be compatible with PHP 8.0 version or superior.
+last_updated: November 14, 2022
+template: concept-topic-template
+---
+
+In the coming weeks, Spryker will release a new version of its demo shops requiring PHP 8.0 as the minimum version - support for the PHP 7.4 version will be stopped. Spryker's new module releases will only be compatible with PHP 8.0 version or superior.
+
+## Impacts
+
+We broke no backward compatibility. If your project followed our advice and requirements in the past twelve months, you will experience no upgradeability issues.
+
+## Migration steps
+
+### Step 1
+
+Update your modules manually in `composer.json`.
+Please use major lock `^` or minor lock `~` if you have changes on the project level or not.
```

Comment:
do you mean "irrespective of whether you have changes on the project level or not?"

Reply:
maybe clarify that this is specific to those modules
```suggestion
Please use major lock `^` or minor lock `~` if you have changes on the project level for respective module constraints.
```